core/entity.ts

```diff
@@ -22,6 +24,7 @@ export function destroy_entity(world: World, entity: Entity) {
         }
     }
     world.Signature[entity] = 0;
+    world.Graveyard.push(entity);
```

Comment:
Maybe you could use a check (in debug mode) to make sure that an entity isn't pushed twice.

Reply:
Good idea, done!